### ISSUE TITLE
Button Reset World

### ISSUE THREAD

**Issue by Arutosio** [modernuo/ModernUO#440]
can you add a command or button inside admin gump where i can reset everything and reset the world as a new world with no items ... to clean up the world (this command can will be used by the owner)

**Comment by kamronbatman** [comment]
Any exceptions? Should delete players too?

**Comment by Arutosio** [comment]
> Any exceptions? Should delete players too?

maybe it would be good to also have a button that eliminates all the banks of all and another that eliminates all the player type characters

**Comment by kamronbatman** [comment]
This will be very dangerous for production servers. Do you have any ideas for how guarding against accidental destruction of the server?

**Comment by Arutosio** [comment]
> This will be very dangerous for production servers. Do you have any ideas for how guarding against accidental destruction of the server?

after pressing the button it asks you to type yes to confirm, and a full backup is made before continuing. if the buckup fails, the restoration must not take place.
the backup server to be able to return to the way it was before doing the restore

**Comment by nstroupr** [comment]
It is my personal opinion that having any option to wipe the world from in-game is dangerous and should be avoided. If an admin account is ever compromised (and I have witnessed and heard about such events), they can wipe everything without having access to the host machine.

**Comment by kamronbatman** [comment]
Yeah, we don't have a concept of dev vs prod server. We can add dev-only mechanics which simply wouldn't be possible in production. So if you wanted to wipe your shard, you would need to load up the server on dev mode and do it.

Thoughts on how to detect, beyond a config flag, when the server is in dev mode?

**Comment by jwvalentine** [comment]
> Yeah, we don't have a concept of dev vs prod server. We can add dev-only mechanics which simply wouldn't be possible in production. So if you wanted to wipe your shard, you would need to load up the server on dev mode and do it.
> 
> Thoughts on how to detect, beyond a config flag, when the server is in dev mode?

Counter question:
Why does it need to be more complicated than a config flag in appSettings or someServerConfig.json?  

Let's assume is a production server:  

- The person would have to go to the config and change it to dev mode. 
- Server would have to restart.
- User would have to log in to a privileged account (Developer or above?) 
- They would need to issue a command - I wouldn't recommend a gump button as the starting point so it cant be "accidentally" clicked.
- A popup gump can be displayed "Hey, if you're really sure you want to delete everything and restart the world from scratch, type the following string: "I WANT TO DELETE ALL PLAYER DATA AND RESET THIS SERVER"

That's a lot of effort which more than adequately addresses danger levels - backups/archives don't need to be tested here, imo.
